djstripe/sync.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
```

Comment:
full module string please

Authors are me and pydanny
https://github.com/kavdev/dj-stripe/blame/%23162-api-updates-through_2015-07-28/djstripe/sync.py